### DISCUSSION TITLE
Chrome errs with unexpected token.

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#11607] [Help] [answered]
See screenshot:
<img width="633" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/107285219/bdab8a7d-0c98-4f7e-803f-7ebf3463e6b2">

The main.css is:
```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

The generated css is:
https://bpa.st/YJXAO#1L8.    It errs at Line 8.

**Comment by wongjn** [comment, marked as answer]
Consider checking you're including the CSS as a stylesheet and not as a JavaScript file.

**Reply by ghost** [reply]
Thanks!
